.github/workflows/frontend.deploy.docs.yml

```diff
@@ -7,15 +7,20 @@ jobs:
   build-and-deploy:
     runs-on: ubuntu-latest
     steps:
-    - name: Checkout
-      uses: actions/checkout@master
+      - name: Set Node version
+        uses: actions/setup-node@v4
+        with:
+          node-version: '>=21'
```

Comment:
did you test this? There was node version problems with vuepress-deploy in the past.